vite.config.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="vitest/config" />
+
+import { defineConfig } from 'vite';
```

Comment:
```suggestion
import { defineConfig } from 'vitest';
```

We don't have `vite` in our `package.json` so we shouldn't directly depend on it.
Afaik, `vitest` exports the `defineConfig` and forwards everything down to `vite`.

Reply:
Fixed!